pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -1250,23 +1252,79 @@ func TestFitScore(t *testing.T) {
 			},
 			runPreScore: true,
 		},
+		{
+			name:         "test case for ScoringStrategy LeastAllocated with Extended resources",
```

Comment:
nit: add 'if PreScore is called' to the name above to highlight the difference between this and the following test case

Reply:
I followed the approach used in the rest of test cases. They only add "if PreScore is not called", assuming that it's called by default:

```
$ grep -i 'PreScore.*called' pkg/scheduler/framework/plugins/noderesources/fit_test.go
                        name: "test case for ScoringStrategy RequestedToCapacityRatio case1 if PreScore is not called",
                        name: "test case for ScoringStrategy MostAllocated if PreScore is not called",
                        name: "test case for ScoringStrategy LeastAllocated if PreScore is not called",
                        name:         "test case for ScoringStrategy LeastAllocated with Extended resources if PreScore is not called",
```

I'm not sure I need to break this approach.